### DISCUSSION TITLE
bug report: Mac OS 15(1.1) key swap, karabiner 15.3.0

### DISCUSSION THREAD

**Discussion by nik123123** [pqrs-org/Karabiner-Elements#4034] [Q&A]
since os 15 the keys <> and °^are swapped. so the other way round. i use a durgod iso keyboard.
can you fix it?!

**Comment by hanche** [comment]
I see the same problem on a Norwegian keyboard. (Magic keyboard, vendor/product id `0x004c/0x029c`. Haven't tried other keyboards yet.) The swapped keys have keycodes 100 and 53. I swap them back using a complex mod, but it's a bit of a pain because I have to repeat the swap with every combination of modifier keys.

**Reply by ElOil** [reply]
Same here - german Keyboard. Please fix this!
